src/accessors/external_accessor.ts

```diff
@@ -591,6 +591,22 @@ export class ExternalAccessor implements IManagementApiAccessor, IManagementSock
     await this.httpClient.post(url, body, requestAuthHeaders);
   }
 
+  // FlowNodeInstances
+  public async getFlowNodeInstancesForProcessInstance(
+    identity: IIdentity,
+    processInstanceId: string,
+  ): Promise<Array<DataModels.FlowNodes.FlowNodeInstance>> {
```

Comment:
With regards to my remarks [here](https://github.com/process-engine/management_api_contracts/pull/50)

```suggestion
  ): Promise<Array<DataModels.FlowNodeInstances.FlowNodeInstance>> {
```